src/global/signal_handler.cc

```diff
@@ -152,6 +158,29 @@ void install_standard_sighandlers(void)
 #include "common/Thread.h"
 #include <errno.h>
 
+void getNameByPid(pid_t pid, char *signal_name) {
+  char proc_pid_path[BUF_SIZE];
+  char buf[BUF_SIZE];
+  sprintf(proc_pid_path, "/proc/%d/status", pid);
```

Comment:
@liewegas  Sir. The unittest_signals  has passed as below in my env.

[==========] Running 6 tests from 3 test cases.
[----------] Global test environment set-up.
[----------] 2 tests from SignalApi
[ RUN      ] SignalApi.SimpleInstall
[       OK ] SignalApi.SimpleInstall (0 ms)
[ RUN      ] SignalApi.SimpleInstallAndTest
[       OK ] SignalApi.SimpleInstallAndTest (0 ms)
[----------] 2 tests from SignalApi (0 ms total)

[----------] 1 test from SignalEffects
[ RUN      ] SignalEffects.ErrnoTest1
[       OK ] SignalEffects.ErrnoTest1 (0 ms)
[----------] 1 test from SignalEffects (0 ms total)

[----------] 3 tests from SignalHandler
[ RUN      ] SignalHandler.Single
2016-05-16 10:12:36.955637 7fefe0edc700 -1  received  signal User defined signal 1 from ./unittest_signals
2016-05-16 10:12:36.955653 7fefe0edc700 -1 ./unittest_signals PID: 18886UID: 0
[       OK ] SignalHandler.Single (1000 ms)
[ RUN      ] SignalHandler.Multiple
2016-05-16 10:12:37.955916 7fefe0edc700 -1  received  signal User defined signal 1 from ./unittest_signals
2016-05-16 10:12:37.955925 7fefe0edc700 -1 ./unittest_signals PID: 18886UID: 0
2016-05-16 10:12:37.956197 7fefe0edc700 -1  received  signal User defined signal 2 from ./unittest_signals
2016-05-16 10:12:37.956204 7fefe0edc700 -1 ./unittest_signals PID: 18886UID: 0
[       OK ] SignalHandler.Multiple (1001 ms)
[ RUN      ] SignalHandler.LogInternal

[WARNING] ./src/gtest-death-test.cc:825:: Death tests use fork(), which is unsafe particularly in a threaded context. For this test, Google Test couldn't detect the number of threads.
[       OK ] SignalHandler.LogInternal (1 ms)
[----------] 3 tests from SignalHandler (2002 ms total)

[----------] Global test environment tear-down
[==========] 6 tests from 3 test cases ran. (2002 ms total)
[  PASSED  ] 6 tests.